src/app/content/components/Page/PageComponent.tsx

```diff
@@ -130,9 +131,16 @@ export default class PageComponent extends Component<PagePropTypes> {
 
   public render() {
     const pageIsReady = this.props.page && this.props.textSize !== null;
+    const PT = this.props.toastTop ?
+      styled(PageToasts)`
+        top:${this.props.toastTop}rem;
+      ` : PageToasts
+    ;
```

Comment:
Usually creating a styled component within another component logs a console warning, you may want to move this outside of PageComponent.